iframe.html

```diff
@@ -13,62 +13,66 @@
 <body style="padding:0; margin:0; width:100%; height:100%; overflow:hidden">
   <div id='container' style='width:100%; height:100%;'></div>
   <script>
-    var queryParams = window.location.href;
-    var playerParam = {
-      "pcode": queryParams.match(/pcode=([^&]*)/)[1],
-      "playerBrandingId": queryParams.match(/pbid=([^&]*)/)[1],
-      "skin": {
-        "config": "",
-        "inline": {"shareScreen": {"embed": {"source": "<iframe width='640' height='480' frameborder='0' allowfullscreen src='//player.ooyala.com/static/v4/stable/latest/skin-plugin/iframe.html?ec=<ASSET_ID>&pbid=<PLAYER_ID>&pcode=<PUBLISHER_ID>'></iframe>"}}}
-      }
-    };
-
-    // Parse docUrl
-    var docUrl = undefined;
-    if (!!queryParams.match(/docUrl=([^&]*)/)) {
-      playerParam["docUrl"] = decodeURIComponent(queryParams.match(/docUrl=([^&]*)/)[1]);
-    } else if (document.referrer) {
-      playerParam["docUrl"] = document.referrer;
-    } else {
-      console.log("iframe.html - document.referrer was not set");
-    }
+    function queryStringToJSON(qs) {
+      var pairs = qs.split('&');
+      var result = {};
+      pairs.forEach(function(pair) {
+        var pair = pair.split('=');
```

Comment:
should `var pair` have another name? the argument to the function is also `pair`